examples/wasm-smith.rs

```diff
@@ -0,0 +1,13 @@
+use arbitrary::Unstructured;
+use wasm_smith::{DefaultConfig, Module};
+
+fn test_wasm_smith() {
+    let seed = "W3B4553MB1Y!!!!!!!!!!!!!!!!!!!!!!!!!!";
+    let mut u = Unstructured::new(seed.as_bytes());
+    if let Ok(module) = Module::new(DefaultConfig::default(), &mut u) {
+        let wasm_buffer = module.to_bytes();
+    }
+}
+
+// Suppress build errors
+fn main() {}
```

Comment:
In general Rust-based examples typically are executable programs which produce some form of output. Could the above get folded into this `main` function and could the input perhaps get printed with the `wasmprinter` crate for a runnable example?